ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/launcher/AbstractPluginBlock.java

```diff
@@ -1069,21 +1069,10 @@ protected void setVisible(boolean visible) {
 	protected abstract LaunchValidationOperation createValidationOperation() throws CoreException;
 
 	/**
-	 * called before the TreeView is refreshed. This allows any subclasses to cache
-	 * any information in the view and redisplay after the refresh.  This is used by the
-	 * OSGiBundleBlock to cache the values of the default launch and auto launch columns
-	 * in the table tree.
+	 * Disposing the editor in tree viewer
 	 *
-	 * @param treeView The tree view that will be refreshed.
 	 */
 	protected void refreshTreeView(CheckboxTreeViewer treeView) {
-		// Remove any selection
-		if (treeView.getTree().getItemCount() > 0) {
-			treeView.getTree().setSelection(treeView.getTree().getItem(0));
-		} else {
-			treeView.setSelection(new StructuredSelection(StructuredSelection.EMPTY));
-		}
-
 		// Reset any editors on the tree viewer
```

Comment:
@Dinesh0723 - can you explain the reason for this change - removal of the above code? for me, this looks un-necessary, but pls convince me!